<launch>
<arg name="ncounts" />
<param name="ncounts" value="$(arg ncounts)" />
<include file="$(find rosplan_stage_demo)/launch/empty_stage_single_robot.launch" />
<include file="$(find ros_vosk)/launch/ros_vosk.launch" />
<node pkg="second_coursework" type="move_robot.py" name="move_robot" output="screen" />
<node pkg="second_coursework" type="video_detection.py"  name="detect_frame" />
<node pkg="second_coursework" type="main.py" name="main" output="screen" />
<node pkg="second_coursework" type="room_check_server.py" name="room_check" output="screen" />
<arg name="vid_folder" default="/home/rm/Videos/cw_videos"/>
<node pkg="second_coursework" type="itr_cw_video.pyc" args="$(arg vid_folder) test" name="itr_cw_video" output="screen" />
</launch>